LocalSettings.php

```diff
@@ -136,6 +136,30 @@
 		'default' => false,
 		'metawiki' => true,
 	),
+	'wgAbuseFilterAvailableActions' => array(
+		'default' => array(
+			 'flag' => true,
```

Comment:
'flag' is tabbed wrongly it is ahead of the others. like
 'flag' instead of
'flag'